notebook2/notebook/static/styles/variables/margins.scss

```diff
@@ -0,0 +1 @@
+$margin: 0.875rem;
```

Comment:
What's the thinking on defining margin globally in-terms of the root element's em? I think of margins as very UI-element-specfic.

Reply:
Every UI element should use a multiple of this value. We need consistent elements across pages.